### DISCUSSION TITLE
Problem with Web accesibility on Prestashop 1.7.9 classic theme

### DISCUSSION THREAD

**Discussion by DPStudioGR** [PrestaShop/PrestaShop#33814] [Q&A]
Hi there ....
I am in great need of help ... I have created an eshop for my business and lately in order to collaborate with a group i am asked to pass WAG (Web accecibility) My problem is that although i have resolved most of the issues on home page, categoty and product pages there are issues on cart, order summary and account creation...
1) On cart page i get a label error on qualtity arrows  as seen on image below and on order account creation  on customer's info fields  again the same...
Has anyone dealt with similar issues or has a sollution ?

Thank you in advance..

![Στιγμιότυπο οθόνης 2023-08-30 132752](https://github.com/PrestaShop/PrestaShop/assets/53015902/11a75907-4cf3-4028-9c66-d741a5fff811)
1
![2a](https://github.com/PrestaShop/PrestaShop/assets/53015902/a324ea0a-2928-4f38-9d57-892b1caa3d6d)

**Comment by kpodemski** [comment]
Hello @Stampa-Stampa 

I didn't have to deal with such requirements, but since the theme's code is open, you can change whatever item you need. That being said, your issues are not really PrestaShop-related. It's not "how PrestaShop does things". It's how the tool to measure accessibility expects them to be and how they should be done to meet the criteria. Maybe you could ask some accessibility specialists 🤔
